hbase-common/src/main/java/org/apache/hadoop/hbase/util/GsonUtil.java

```diff
@@ -43,6 +44,8 @@ private GsonUtil() {
    */
   public static GsonBuilder createGson() {
     return new GsonBuilder().setLongSerializationPolicy(LongSerializationPolicy.STRING)
+      .excludeFieldsWithModifiers(Modifier.TRANSIENT).excludeFieldsWithModifiers(Modifier.STATIC)
```

Comment:
Do we need to exclude static and private too?

Reply:
Ah, I see it now. You're right.